geoportal/Dockerfile

```diff
@@ -45,17 +46,22 @@ RUN node --version
 RUN npm install --no-optional && npm cache clear --force
 RUN /etc/apiv4/rebuild_api.sh
 
+COPY . /app
 WORKDIR /app
 # sad fix, to allow webpack's file-loader to find files with query string & hash added
 RUN ln -s /usr/lib/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.eot /usr/lib/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.eot?#iefix && \
     ln -s /usr/lib/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.svg /usr/lib/node_modules/@fortawesome/fontawesome-free/webfonts/fa-regular-400.svg#fontawesome && \
     ln -s /usr/lib/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.eot /usr/lib/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.eot?#iefix && \
     ln -s /usr/lib/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.svg /usr/lib/node_modules/@fortawesome/fontawesome-free/webfonts/fa-solid-900.svg#fontawesome
 
-
 # RUN make checks
 RUN make build
 
+RUN curl -L https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/releases/download/$(cat v4_release.txt)/bundle.tgz > /tmp/bundle.tgz && \
+    tar -xzvf /tmp/bundle.tgz -C /tmp && \
+    mkdir -p /usr/lib/node_modules/luxembourg-geoportail && \
+    mv /tmp/bundle/ /usr/lib/node_modules/luxembourg-geoportail
```

Comment:
I'm not a big fan of manipulating the npm dependencies "manually" here, as the `package.json` is already there to manage such things. Maybe we could find a way to continue making the bundle available that is directly accessible for the `package.json` (in a dedicated branch or via npm for example).

Reply:
I agree with you.
There are three arguments in favor of this approach, though:
- it's "less bad" than before with the bundle inside the repo and the whole source files installed and unused
- it's only for the time of the backport while v3 and v4 coexist. And it's by far not the only hack for this time
- in case of a V4 change, the docker image only has to be rebuilt for the last stages, the (long) installation of basic V3 npm dependencies can be skipped

so in the end I do not feel too bad with this "hack", but I'll check if we can build a "fake" npm package to install via the git hash...

Reply:
Agree with @tkohr